### DISCUSSION TITLE
Can't link to Akeeba Backup at WordPress

### DISCUSSION THREAD

**Discussion by rvalitov** [akeeba/panopticon#805] [Q&A (Community Support)]
I tried WordPress version 5.3, 5.6 and 5.7. All 3 installations do not link to Akeeba Backup. Tried to reinstall Backup and connector - nothing changes. Could it be that there's a minimum version requirement for WordPress or Akeeba Backup?

**Comment by nikosdion** [comment]
Most importantly, which version of Akeeba Backup are you using on each of these sites?

**Reply by rvalitov** [reply]
7.9.2 on all websites

**Reply by nikosdion** [reply]
Right now you won't be able to use it. I had to make a few changes in both the connector and Panopticon itself. I plan to make a new release next week which will address these issues.
